### PR TITLE
LPS-182384 move portal-upload to modules part One

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, 

This is the clean up for directly using UploadServletRequestImpl() and UploadPortletRequestImpl() in the production code and test.
Thanks for checking

### CODE REVIEW

File: modules/apps/portal/portal-upload-test-util/src/main/java/com/liferay/portal/upload/test/util/UploadTestUtil.java [hunk 1]
```diff
@@ -16,19 +16,31 @@
...
+import javax.portlet.PortletRequest;
+
 import javax.servlet.http.HttpServletRequest;
 
 /**
  * @author Jiefeng Wu
  */
 public class UploadTestUtil {
 
+	public static UploadPortletRequest createUploadPortletRequest(
```
Comment (jeffwu0724) [inline modules/apps/portal/portal-upload-test-util/src/main/java/com/liferay/portal/upload/test/util/UploadTestUtil.java:36]: The reason I added this method is, I would like to centralize the logic of using new UploadPortletRequestImpl() in the test.
Later when I moved `UploadPortletRequestImpl()` to modules, I will use `BundleWiring Bundles ` to get the logic of Impl as Dante advised, and I will replace new UploadPortletRequestImpl() to using Bundle in this code

**Comment by Kvale1733** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:echo

**Comment by zhangyan0701** [comment]
ci:test:lima

**Comment by zhangyan0701** [comment]
ci:test:sf

**Comment by zhangyan0701** [comment]
The SF failure should be related:
`Library 'org.springframework:spring-web:5.2.24.RELEASE' contains known vulnerabilities(Pivotal Spring Framework contains unsafe Java deserialization methods, https://github.com/advisories/GHSA-4wrc-f8pq-fpqp): ./modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-container-test/build.gradle (SourceCheck:LibraryVulnerabilitiesCheck)`

Full console log:
https://test-1-40.liferay.com/job/test-portal-source-format/1754/consoleFull

**Comment by jeffwu0724** [comment]
@zhangyan0701 

Seems like this SF issue was reported before https://issues.liferay.com/browse/LPS-187249, and it is not caused by this ticket. 
And if I run "ant format-source-current-branch", it will run successfully.
@Kvale1733 Could you please confirm this for me?

Thanks!

**Comment by Kvale1733** [comment]
@jeffwu0724 Yes I've seen that same sf issue on another PR too. It's unrelated. I'll check the other results and pass if there is no issue.

**Comment by Kvale1733** [comment]
@jeffwu0724 Regressions found. 

1) com.liferay.journal.web.internal.portlet.action.test.ImportDataDefinitionMVCActionCommandTest
```
exec] com.liferay.journal.web.internal.portlet.action.test.ImportDataDefinitionMVCActionCommandTest > testDataLayoutFieldNamesAreEqualToDataDefinitionFieldNames FAILED
     [exec]     java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
     [exec]     	at java.util.ArrayList.rangeCheck(ArrayList.java:653)
     [exec]     	at java.util.ArrayList.get(ArrayList.java:429)
     [exec]     	at com.liferay.journal.web.internal.portlet.action.test.ImportDataDefinitionMVCActionCommandTest._getImportedDataDefinition(ImportDataDefinitionMVCActionCommandTest.java:309)
     [exec]     	at com.liferay.journal.web.internal.portlet.action.test.ImportDataDefinitionMVCActionCommandTest.testDataLayoutFieldNamesAreEqualToDataDefinitionFieldNames(ImportDataDefinitionMVCActionCommandTest.java:82)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
     [exec]     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
     [exec]     	at java.lang.reflect.Method.invoke(Method.java:498)
     [exec]     	at com.liferay.arquillian.extension.junit.bridge.server.TestExecutorRunnable$3.evaluate(TestExecutorRunnable.java:353)
     [exec]     	at com.liferay.portal.kernel.test.rule.AbstractTestRule$2.evaluate(AbstractTestRule.java:99)
```
[Log](https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3336/modules-integration-mysql57-jdk8/0/12/jenkins-console.txt.gz?authuser=0)

2) com.liferay.info.request.struts.test.AddInfoItemStrutsActionTest
```
[exec] com.liferay.info.request.struts.test.AddInfoItemStrutsActionTest > testAddInfoItemInvalidBigDecimalTooBig STARTED
[beanshell] 2023-06-15 18:41:59.410 WARN  [com.liferay.info.test-executor-thread][UploadServletRequestImpl:192] Unable to parse upload request: the request was rejected because no multipart boundary was found
     [exec] 
     [exec] com.liferay.info.request.struts.test.AddInfoItemStrutsActionTest > testAddInfoItemInvalidBigDecimalTooBig FAILED
     [exec]     com.liferay.portal.kernel.events.ActionException: java.lang.NullPointerException
     [exec]     	at com.liferay.portal.events.ServicePreAction.run(ServicePreAction.java:160)
     [exec]     	at com.liferay.portal.kernel.events.Action.processLifecycleEvent(Action.java:34)
     [exec]     	at com.liferay.portal.events.EventsProcessorUtil.process(EventsProcessorUtil.java:80)
```
[Log](https://testray.liferay.com/reports/production/logs/2023-06/test-1-40/test-portal-acceptance-pullrequest(master)/3336/modules-integration-mysql57-jdk8/0/7/jenkins-console.txt.gz?authuser=0)

3) com.liferay.template.web.internal.portlet.action.test.UpdateTemplateEntryMVCActionCommandTest
```
 [exec] com.liferay.template.web.internal.portlet.action.test.UpdateTemplateEntryMVCActionCommandTest > testUpdateTemplateEntry FAILED
     [exec]     java.lang.RuntimeException: Unable to unwrap the portlet request from class com.liferay.portal.kernel.test.portlet.MockLiferayPortletActionRequest
     [exec]     	at com.liferay.portlet.LiferayPortletUtil.getLiferayPortletRequest(LiferayPortletUtil.java:49)
     [exec]     	at com.liferay.portal.util.PortalImpl.getUploadPortletRequest(PortalImpl.java:5545)
     [exec]     	at com.liferay.template.web.internal.portlet.action.UpdateTemplateEntryMVCActionCommand.doTransactionalCommand(UpdateTemplateEntryMVCActionCommand.java:68)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
     [exec]     	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
```
[Log](https://testray.liferay.com/reports/production/logs/2023-06/test-1-29/test-portal-acceptance-pullrequest(master)/7466/modules-integration-mysql57-jdk8/1/4/jenkins-console.txt.gz?authuser=0)

4) com.liferay.object.web.internal.portlet.action.test.ExportImportObjectDefinitionTest
```
  [exec] com.liferay.object.web.internal.info.item.provider.test.ObjectEntryInfoItemFormProviderTest > testObjectEntryInfoItemFormProvider STARTED
     [exec] 
     [exec] com.liferay.object.web.internal.info.item.provider.test.ObjectEntryInfoItemFormProviderTest > testObjectEntryInfoItemFormProvider FAILED
     [exec]     java.lang.IllegalStateException: Permission checker is null
     [exec]     	at com.liferay.portal.security.permission.internal.InlineSQLHelperImpl.isEnabled(InlineSQLHelperImpl.java:125)
     [exec]     	at com.liferay.portal.kernel.security.permission.InlineSQLHelperUtil.isEnabled(InlineSQLHelperUtil.java:41)
```
[Log](https://testray.liferay.com/reports/production/logs/2023-06/test-1-37/test-portal-acceptance-pullrequest(master)/12362/modules-integration-mysql57-jdk8/0/19/jenkins-console.txt.gz?authuser=0)

**Pull request closed without merging**